### ISSUE TITLE
 Luminosity ratio of Editor buttons border is less than 3:1 against its background color in ckeditor tool tip bar

### ISSUE THREAD

**Issue by mohitrock1** [ckeditor/ckeditor4#3932]
**User Experience:**
User might get affected if control border contrast ratio is less than 3:1 against its background color and user will not be able to identify the control while navigating on screen.

**Test Environment:**
#URL:  https://ckeditor.com/ckeditor-4/
Tools: Accessibility insight tool
![image](https://user-images.githubusercontent.com/29257206/76758437-b01f4b00-67af-11ea-8959-4315dff1c295.png)

**Repro Steps:**

1. Open URL: https://ckeditor.com/ckeditor-4/
2. Navigate to Accessibility Checker
3. Click on Demo.
4. Verify the issue

**Actual Result:**
Luminosity ratio of Editor buttons border on  is less than 3:1 against its background color.

**Expected Result:**
Luminosity ratio of Editor buttons border should be greater  than 3:1 against its background color.

**MAS Reference:**
MAS 1.4.11 - Non-text Contrast

**Reference Links**
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by Dumluregn** [comment]
Hello, according to [WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html):

> Boundaries
This success criteria does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed.

So I'm closing this issue as CKEditor 4 is WCAG 2.1-compliant.
